_posts/2024-04-20-16.0.0-release.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: post
+title: "Apache Arrow 16.0.0 Release"
+date: "2024-04-20 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 16.0.0 release. This covers
+over 3 months of development work and includes [**385 resolved issues**][1]
+on [**586 distinct commits**][2] from [**119 distinct contributors**][2].
+See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 15.0.0 release, Jeffrey Vo, Jay Zhan, Bryce Mecum and Sarah Gilmore
+have been invited to be committers.
```

Comment:
Joel Lubinitsky? There was [a thread on the mailing list](https://lists.apache.org/thread/4w6n6wgfz1f9zq0jmvj62zjst0gx76m1) but he doesn't show up on the committers' list.

Reply:
Good catch @CurtHagenlocher. Joel is currently [listed as a committer](https://people.apache.org/committer-index.html) in the ASF database, they [accepted on April 1](https://lists.apache.org/thread/4w6n6wgfz1f9zq0jmvj62zjst0gx76m1), and the release for 16.0.0 RC0 wasn't cut until [April 17](https://lists.apache.org/thread/qwy525yttrgjnl07cbqj6l03rznh8864) so I think this is the correct release to include that in.

Reply:
Change suggested in https://github.com/apache/arrow-site/pull/500/files#r1581744512. cc @raulcd

Reply:
yes, it seems I forgot to check the April mailing list announcements! Thanks for catching it.